### PR TITLE
install/uninstall with bash_completion

### PR DESCRIPTION
**Pull request by stevemao**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -23,6 +23,9 @@ $(module)/talk.html: $(module)/talk.rst
 
 install: $(BIN)
 	install $(BIN) $(PREFIX)/bin
+	@mkdir -p /etc/bash_completion.d
```
Comment (lucaswerkmeister) [inline Makefile]: Does it really make sense to create this folder? If it’s not already there, then the system probably stores its bash completion scripts elsewhere, and this folder probably won’t be sourced.

Comment (stevemao) [inline Makefile]: True. But I basically copied it from https://github.com/tj/git-extras/blob/master/Makefile#L44

**Comment by mhagger** [comment]
Given that our bash completion relies on Git's bash completion already being installed, I'd worry a little bit that this would install broken software on the user's system. What happens if we go through with this installation but Git's isn't installed? Does it fail gracefully, or (for example) does it make the user's shell unusable?

@ralfth, do you have an opinion about this PR?

**Comment by ralfth** [comment]
The function for bash completing imerge is executed by Git's completion script, so I think we're save.
I tried it out, anyway. I removed Git's completion locally and didn't see any error because functions that are using (missing) functions from Git's completion script weren't executed.

**Pull request merged**